docs/manage/idp/azure-ad.md

```diff
@@ -56,4 +51,4 @@ infra providers add azure \
      - **User** > **User.Read** - Click **Add permissions**.
      - Click **Grant admin consent for Default Directory** and select **Yes** when prompted.
        ![Add API permissions](../../images/azure-setup/connect-users-azure-6.png)
-9. From the **Overview** tab copy the **Application (client) ID**, **Directory (tenant) ID**, and **Client Secret** values and provide them into Infra's Dashboard or CLI.
+8. From the **Overview** tab copy the **Application (client) ID**, **Directory (tenant) ID**, and **Client Secret** values and provide them into Infra's Dashboard or CLI.
```

Comment:
For markdown, I suggest just using `1.` for order lists since it will automatically get converted to the right number. Leaving it as `1.` reduces diffs like this as well as mental gymnastics of maintaining the ordering